include/reader.h

```diff
@@ -0,0 +1,42 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        :::      ::::::::   */
+/*   reader.h                                           :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: lporoshi <lporoshi@student.42.fr>          +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2023/12/09 16:48:50 by lporoshi          #+#    #+#             */
+/*   Updated: 2023/12/09 16:53:27 by lporoshi         ###   ########.fr       */
+/*                                                                            */
+/* ************************************************************************** */
+
+#ifndef READER_H
+# define READER_H
+
+# include <stdio.h>
+# include <stdlib.h>
+# include <readline/readline.h>
+# include <readline/history.h>
+
```

Comment:
move includes to reader.c
src: https://google.github.io/styleguide/cppguide.html#Include_What_You_Use

Reply:
done